connect/src/com/telenor/mobileconnect/MobileConnectSdkProfile.java

```diff
@@ -184,17 +133,17 @@ private void tryMccMnc(
                 operatorDiscoveryConfig.getOperatorDiscoveryRedirectUri(),
                 mcc,
                 mnc,
-                odCallbackForMccMnc);
-    }
+                new Callback<OperatorDiscoveryAPI.OperatorDiscoveryResult>() {
```

Comment:
Yeah I meant referencing them.
But anyway, now that you have also changed the .well-known fetching to happen first before proceeding to the authorize (it's not in parallel anymore), I think it'll be better to construct some WellKnown immutable struct and pass it to the Authorization/WebView. That way you won't rely on the volatile references and the model is easier to understand I believe.